Lombiq.Hosting.Tenants.Admin.Login/Views/TenantAdminShape.cshtml

```diff
@@ -1,5 +1,5 @@
-@using Microsoft.AspNetCore.Http;
-@inject Lombiq.Hosting.Tenants.Admin.Login.Services.ITenantLoginPasswordValidator TenantLoginPasswordProvider;
+@using Microsoft.AspNetCore.Http
+@inject Services.ITenantLoginPasswordValidator TenantLoginPasswordProvider
```

Comment:
I think the OSOCE build fails because you removed `Lombiq.Hosting.Tenants`: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/4707574849/jobs/8349509975?pr=449#step:6:111

Reply:
That's sad. I forgot to check the OSOCE build, tbh.